Modules/System/WebDAV/src/WebDAVClient.Codeunit.al

```diff
@@ -0,0 +1,84 @@
+codeunit 5678 "WebDAV Client"
```

Comment:
This is the facade, so xml documentation would need to be added here. (Also MIT stuff)

Reply:
XML documentation and MIT headers added